components/insight/SRC/org/openmicroscopy/shoola/util/roi/figures/MeasureLineFigure.java

```diff
@@ -828,7 +831,8 @@ public void setStartPoint(Point2D.Double p)
 	 */
 	public int splitSegment(Point2D.Double split) 
 	{
-		if (isReadOnly() || !interactable) return -1;
+		if (isReadOnly() || !interactable || !allowSplitSegment)
```

Comment:
maybe having a private method ``canBemodified`` will be more suitable
if case we add another constaint